src/main/java/com/google/codeu/servlets/UsersServlet.java

```diff
@@ -10,6 +10,11 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
```

Comment:
This didn't merge quite right, we shouldn't have "user service" and "user service factory" in this file, that was removed in a recent change